conf/zapiperf/7mode/8.2.5/volume_node.yaml

```diff
@@ -1,8 +1,8 @@
 
 name:                     VolumeNode
 query:                    volume
-object:                   volume
-
+object:                   node_vol
```

Comment:
where is node_vol used?

Reply:
no where yet.  This template is not utilized in current dashboards